app/src/main/java/org/mozilla/fenix/library/recentlyclosed/RecentlyClosedFragment.kt

```diff
@@ -30,33 +32,63 @@ import org.mozilla.fenix.components.FenixSnackbar
 import org.mozilla.fenix.components.StoreProvider
 import org.mozilla.fenix.ext.getRootView
 import org.mozilla.fenix.ext.requireComponents
+import org.mozilla.fenix.ext.setTextColor
 import org.mozilla.fenix.ext.showToolbar
 import org.mozilla.fenix.library.LibraryPageFragment
 
 @Suppress("TooManyFunctions")
-class RecentlyClosedFragment : LibraryPageFragment<RecoverableTab>() {
+class RecentlyClosedFragment : LibraryPageFragment<RecoverableTab>(), UserInteractionHandler {
     private lateinit var recentlyClosedFragmentStore: RecentlyClosedFragmentStore
     private var _recentlyClosedFragmentView: RecentlyClosedFragmentView? = null
     protected val recentlyClosedFragmentView: RecentlyClosedFragmentView
         get() = _recentlyClosedFragmentView!!
 
     private lateinit var recentlyClosedInteractor: RecentlyClosedFragmentInteractor
+    private lateinit var recentlyClosedController: RecentlyClosedController
 
     override fun onResume() {
         super.onResume()
         showToolbar(getString(R.string.library_recently_closed_tabs))
     }
 
     override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
-        inflater.inflate(R.menu.library_menu, menu)
+        if (recentlyClosedFragmentStore.state.selectedTabs.isNotEmpty()) {
```

Comment:
Sorry, I must have missed this in the previous iteration.

I don't think we need to add a menu here since the parent `LibraryPageFragment` handles the overlay that shows up with the menu built in.

In fact, I think we can entirely remove `onCreateOptionsMenu` and `onOptionsItemSelected` since that pattern is inconsistent with our other "library" views.

Reply:
I tried to remove the `onCreateOptionsMenu` and `onOptionsItemSelected` methods, but it didn't work. I have checked `LibraryPageFragment`, from what I can see it doesn't seem to handle the options menu. I also looked at `BookmakrFragment`, `DownloadFragment`, `HistoryFragment` and they all override the two aforementioned methods. 

On the other hand, the `LibraryPageView` does handle updating the toolbar color in selection mode.

Reply:
You're right, it looks like the 'history' view is the one that is inconsistent; that's a separate issue then. 👌